gradle/scripts/dependencyDefinitions.gradle

```diff
@@ -180,13 +180,16 @@ ext.externalDependency = [
     'reactivex': 'io.reactivex.rxjava2:rxjava:2.1.0',
     "slf4j": [
         "org.slf4j:slf4j-api:" + slf4jVersion,
-        "org.slf4j:slf4j-log4j12:" + slf4jVersion,
+        "org.slf4j:log4j-over-slf4j:" + slf4jVersion,
```

Comment:
OK now I understand it through reading this [link](http://www.slf4j.org/legacy.html#:~:text=SLF4J%20ship%20with%20a%20module,simply%20by%20replacing%20the%20log4j.) !